src/System.Diagnostics.Process/System.Diagnostics.Process.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
Is this change necessary?

Reply:
It happened automatically by VS.  I tend to not fight VS, as it is usually a losing battle.